unit_tests/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 #
+#required(VERSION 3.5)
```

Comment:
I'm sorry our current CI adopts cmake 2.8. 
Do you have particular reasons to use 3.5?

Reply:
No special reason, it was the version I had on my computer. 
I'll set it on 2.8... or better not to specify it at all?

Reply:
Then, it is better to remove it :)